From 9ca6d62996ba9e552a825f02d5c6d50bd683c612 Mon Sep 17 00:00:00 2001
From: Camelia Groza <camelia.groza@nxp.com>
Date: Thu, 7 Dec 2017 17:49:34 +0200
Subject: [PATCH 207/767] sdk_dpaa: ceetm: increment the interface's ern
 counters when needed

Signed-off-by: Camelia Groza <camelia.groza@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c
index db61c3a79353..b090db1f2eb8 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_eth_ceetm.c
@@ -86,6 +86,7 @@ static void ceetm_ern(struct qman_portal *portal, struct qman_fq *fq,
 	/* Increment DPA counters */
 	dpa_percpu_priv->stats.tx_dropped++;
 	dpa_percpu_priv->stats.tx_fifo_errors++;
+	count_ern(dpa_percpu_priv, msg);
 
 	/* Increment CEETM counters */
 	cls = ceetm_fq->ceetm_cls;
-- 
2.17.0

